### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by anindya2001**
Replaced the link to my paper 'Moving Object Detection for Event-based Vision using Graph Spectral
Clustering' with the IEEE/CVF Open Access version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -402,8 +402,8 @@ arXiv, 2021.
 *[Standard and Event Cameras Fusion for Feature Tracking](https://dl.acm.org/doi/10.1145/3459066.3459075)*,  
 Int. Conf. on Machine Vision and Applications (ICMVA), 2021. [Code](https://github.com/LarryDong/FusionTracking)
 - <a name="Mondal21iccvw"></a>Mondal, A., Shashant, R., Giraldo, J. H., Bouwmans, T., Chowdhury, A. S.,  
-*[Moving Object Detection for Event-based Vision using Graph Spectral Clustering](https://www.researchgate.net/publication/354462913_Moving_Object_Detection_for_Event-based_Vision_using_Graph_Spectral_Clustering)*,  
-IEEE Int. Conf. Computer Vision Workshop (ICCVW), 2021.
+*[Moving Object Detection for Event-based Vision using Graph Spectral Clustering](https://openaccess.thecvf.com/content/ICCV2021W/GSP-CV/html/Mondal_Moving_Object_Detection_for_Event-Based_Vision_Using_Graph_Spectral_Clustering_ICCVW_2021_paper.html)*,
```
Comment (guillermogb) [inline README.md:405]: Hi. If the html page has the same information as the pdf (i.e., there is no supplementary material), I suggest providing just one link here (to the pdf, like the link below).

**Review by guillermogb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (anindya2001) [reply README.md:405]: Yes, you can provide the link to the PDF only.

Comment (guillermogb) [reply README.md:405]: could you commit the change and then we merge?

File: README.md [hunk 2]
```diff
@@ -402,7 +402,7 @@ arXiv, 2021.
 *[Standard and Event Cameras Fusion for Feature Tracking](https://dl.acm.org/doi/10.1145/3459066.3459075)*,  
 Int. Conf. on Machine Vision and Applications (ICMVA), 2021. [Code](https://github.com/LarryDong/FusionTracking)
 - <a name="Mondal21iccvw"></a>Mondal, A., Shashant, R., Giraldo, J. H., Bouwmans, T., Chowdhury, A. S.,  
-*[Moving Object Detection for Event-based Vision using Graph Spectral Clustering](https://openaccess.thecvf.com/content/ICCV2021W/GSP-CV/html/Mondal_Moving_Object_Detection_for_Event-Based_Vision_Using_Graph_Spectral_Clustering_ICCVW_2021_paper.html)*,  
+*[Moving Object Detection for Event-based Vision using Graph Spectral Clustering](https://openaccess.thecvf.com/content/ICCV2021W/GSP-CV/papers/Mondal_Moving_Object_Detection_for_Event-Based_Vision_Using_Graph_Spectral_Clustering_ICCVW_2021_paper.pdf)*,  
 IEEE Int. Conf. Computer Vision Workshop (ICCVW), 2021. [PDF](https://openaccess.thecvf.com/content/ICCV2021W/GSP-CV/papers/Mondal_Moving_Object_Detection_for_Event-Based_Vision_Using_Graph_Spectral_Clustering_ICCVW_2021_paper.pdf).
```
Comment (guillermogb) [inline README.md:406]: Please delete the 2nd link, since it is the same as the first one.

**Review by guillermogb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**